Description: Fix unintialized warning in nbtdesc
 Provide initial values for variables to silence maybe-uninitialized
 warnings/errors from GCC.
Author: Bradford D. Boyle <bradford.d.boyle@gmail.com>
Forwarded: no
Last-Update: 2023-02-04

--- a/src/backend/access/rmgrdesc/nbtdesc.c
+++ b/src/backend/access/rmgrdesc/nbtdesc.c
@@ -26,9 +26,9 @@
 	char		*ptr;
 	xl_btree_insert	*xlrec = (xl_btree_insert *) rec;
 	xl_btree_metadata *md;
-	BlockNumber	blkno;	
+	BlockNumber	blkno = 0;
 	bool		fullpage;
-	Size		datalen;
+	Size		datalen = 0;
 
 	fullpage = XLogRecHasBlockImage(record, 0);
 	XLogRecGetBlockTag(record, 0, NULL, NULL, &blkno);
